source/_components/plex.markdown

```diff
@@ -161,44 +163,4 @@ Plays a song, playlist, TV episode, or video on a connected client.
 
 ## Sensor
 
-The `plex` sensor platform will monitor activity on a given [Plex Media Server](https://plex.tv/). It will create a sensor that shows the number of currently watching users as the state. If you click the sensor for more details it will show you who is watching what.
-
-If you want to enable the plex sensor, add the following lines to your `configuration.yaml`:
-
-```yaml
-# Example configuration.yaml entry
-sensor:
-  - platform: plex
-```
-
-{% configuration %}
-host:
-  description: The IP address or hostname of your Plex server.
-  required: false
-  default: localhost
-  type: string
-port:
-  description: The port of your Plex Server.
-  required: false
-  default: 32400
-  type: integer
-name:
-  description: Name of the Plex server.
-  required: false
-  default: Plex
-  type: string
-token:
-  description: A valid X-Plex-Token for your Plex server.
-  required: false
-  type: string
-ssl:
-  description: Use HTTPS to connect to Plex server, **NOTE:** host **must not** be an IP when this option is enabled.
-  required: false
-  default: false
-  type: boolean
-verify_ssl:
-  description: Verify the SSL certificate of your Plex server. You may need to disable this check if your local server enforces secure connections with the default certificate.
-  required: false
-  default: true
-  type: boolean
-{% endconfiguration %}
+The `plex` sensor platform will monitor activity on a given [Plex Media Server](https://plex.tv/). It will create a sensor that shows the number of currently watching users as the state. If you click the sensor for more details it will show you who is watching what.
```

Comment:
```suggestion
The `plex` sensor platform will monitor activity on a given [Plex Media Server](https://plex.tv/). It will create a sensor that shows the number of currently watching users as the state. If you click the sensor for more details, it will show you who is watching what.
```